tests/unit/composables/Project.spec.js

```diff
@@ -438,4 +440,11 @@ describe('Test composable: Project', () => {
       expect(GitEvent.PushEvent.next).toBeCalled();
     });
   });
+
+  describe('Test function: gitAdd', () => {
+    it('should call git add', async () => {
+      const result = await gitAdd('projectId', 'filepath');
```

Comment:
...so here there should be no result, so the const is not needed.

Reply:
I did exactly the same as `initProject`

Reply:
replace  by a `toBeCalled` ;)